pyupgrade.py

```diff
@@ -1034,13 +942,23 @@ def visit_ClassDef(self, node):
             ):
                 self.remove_decorators.add(_ast_to_offset(decorator))
 
-        self.generic_visit(node)
+        for i, base in enumerate(node.bases):  # pragma: no cover (py3+)
```

Comment:
hmmmm, this seems not right -- it didn't need this comment on the left, why the change now?

Reply:
Let me scope this down